### DISCUSSION TITLE
Tauri blocks popups from @trezor/connect

### DISCUSSION THREAD

**Discussion by sumi0820** [tauri-apps/tauri#6626] [Q&A]
I'm developing DApp with Tauri and trying to use `@trezor/connect-web`.

However, somehow a popup from `@trezor/connect-web` is blocked when running it as a desktop app while it's properly opened on browsers. (There are no error messages in console as below)

![Screenshot 2023-04-04 at 10 49 50 AM](https://user-images.githubusercontent.com/48553998/229666848-2c4d6ba3-e9c1-4b74-8f46-ba35076c496a.png)

If I try to open a new webview window using `WebviewWindow` api from `@tauri-apps/api/window`, new window can be opened, but I would like to open it from `@trezor/connect-web` API as well. 

The configuration I modified was only `tauri.conf.json`.
```
    "allowlist": {
      "all": true
    },
```

Could you kindly let me know if there's a way to enable popups on Tauri?

**Comment by FabianLars** [comment]
This is tracked here: https://github.com/tauri-apps/tauri/issues/6501. There's no solution for it yet.
